docs/platforms/unreal/migration/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Migration Guide
+description: "Learn more about migrating to the current version."
+sidebar_order: 8000
+---
+
+## Migrating to 1.0.0
+
+### Breaking changes
+
+#### Sentry Classes Instantiation
+
+Sentry class objects created via `NewObject<T>` in C++ or `ConstructObjectFromClass` in Blueprints now require an explicit call to their `Initialize` method (if available) before use. In Blueprints, it's recommended to use the provided Sentry library functions to create these entities as they automatically call `Initialize` and return a fully initialized, ready-to-use object.
+
+#### Cleanup Public Classes
+
+We cleaned up our public API by removing a few functions and classes to streamline the SDK and remove ambiguities. The following changes were made:
+
+- Removed `USentryId` class and replaced its usages with `FString`
+- Removed `USentrySubsystem::ConfigureScope` function
+- Removed `USentryLibrary::StringToBytesArray` function
+- Removed `USentryLibrary::ByteArrayToString` function
+- Removed `USentryLibrary::SaveStringToFile` function
+- Removed `USentryScope::SetEnvironment` function
+- Removed `USentryScope::GetEnvironment` function
+- Removed `USentryScope::SetDist` function
+- Removed `USentryScope::GetDist` function
+
+#### User feedback
+
+We have reworked the user feedback feature so now it no longer needs to be associated with a specific event and the only required input is the text message. If you were using this functionality in your project consider updating it accordingly:
+
+- Replace `USentryUserFeedback` class references with `USentryFeedback`
+- Replace `USentrySubsystem::CaptureUserFeedback` function usages with `USentrySubsystem::CaptureFeedback`
+- Replace `USentryLibrary::CreateSentryUserFeedback` function usages with `USentryLibrary::CreateSentryFeedback`
+
+#### Tracing
+
+The `USentrySubsystem::StartTransactionWithContextAndOptions` function now takes an `FSentryTransactionOptions` struct instead of a `TMap<FString, FString>`, improving clarity and allowing better extensibility in the future.
+
+Also, the `USentrySamplingContext` class now utilizes `FSentryVariant` instead of strings for custom sampling context values.
+
+#### Event ID format (Windows/Linux)
+
+On Windows and Linux, `ToString` function of `SentryId` class now returns the sanitized string that doesn't contain dashes.
+
+#### Global scope configuration
+
+With the removal of `USentrySubsystem::ConfigureScope` global scope values (tags, context, attachments, etc.) should now be configured directly via the subsystem interface.
```

Comment:
Can we have a code snippet as an example of what it used to be before and what has to be now?